### ISSUE TITLE
Export results to Raven

### ISSUE THREAD

**Issue by thyholm1234** [Mattk70/Chirpity-Electron#146]
First of all a huge thanks to you Matthew for making this amazing app! 

I use it alongside Raven - sometimes it finds something I've missed - and sometimes I don't wont to go through a whole night. 

That has made me wonder if it would be possible to export the results to Raven, so they would show up in the sonogram as I manually go through the night. 

I've attached the format Raven uses for it's selections. The idea is that when importing multiple files to Raven - you'd have to use the exact same starting time as Chirpity Reports - and then maybe just have detections show from 0-20 kHz. 

[240919_028_Tr1.Table.1.selections.txt](https://github.com/user-attachments/files/17070608/240919_028_Tr1.Table.1.selections.txt)

In the perfect world it would place the most likely bird as an annotation for Raven.

**Comment by Mattk70** [comment]
Hi and thanks! I am curious as to why you prefer to use Raven to run through manually. You could do that in Chirpity?

**Comment by thyholm1234** [comment]
Hi Matt

The customizability of Raven lets you split the sonogram into 3 bars and adjust how the file is paged. 

Fx.: each bar shows 20 secs and when I skip to the next section - theres a 10% = 6 sec overlap. 

In addition Raven lets you have grids and Hz is shown on the y-axis. (I might have overlooked something as there's a screenshot with the Hz shown in the manual)

I also miss the opportunity to simply drag the sonogram with the cursor. 

If this was introduced I might be able to ditch Raven - but I still think that exporting the detections to Raven would be a great feature. 

Okay - now I'm just coming up with ideas - that have nothing to do with the first.

I don't know if you know [wSNR](https://xeno-canto.org/article/275) - but it would be super awesome if - in advanced settings - it would be able to run the analysis of sound quality and then put it as an post-fix to the name.

**Comment by Mattk70** [comment]
I've added an export to Raven selection table option which you'll find on the file menu in the next release. It generates a single selections text file, but should work with multiple files' analysis.

As I used to have the frequency labels on the spectrgram, I've re-enabled that as an option. I don't like them because the can only be placed over the spec, obscuring a portion, which is why the frequency is now displayed with a tooltip on the cursor. Another issue is the y-axis labels are a little out, but as far as I can make out,  it's an issue with the wavesurfer library and I can't really do anything about it.
